rotkehlchen/tests/api/test_evm_transactions.py

```diff
@@ -89,3 +89,48 @@ def test_query_transactions(rotkehlchen_api_server: 'APIServer'):
         for entry in result['entries']:
             assert entry['entry']['timestamp'] <= last_ts
             last_ts = entry['entry']['timestamp']
+
+
+@pytest.mark.parametrize('optimism_accounts', [[
+    '0xb8553D9ee35dd23BB96fbd679E651B929821969B',
+]])
+@pytest.mark.parametrize('should_mock_price_queries', [True])
+@pytest.mark.parametrize('default_mock_price_value', [FVal(1.5)])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+def test_query_transactions_single_chain(rotkehlchen_api_server: 'APIServer'):
+    """
+    Test that querying the evm transactions endpoint for transactions in a single evm chain
+    works even if the address has transactions in other chains too.
+
+    This test uses real data.
+
+    TODO: Mock network here. Need to mock transaction query for both mainnet and optimism etherscan
+    """
+    # Ask for all evm transactions (test addy has both optimism and mainnet)
```

Comment:
The comment is wrong, you query only ethereum. If you explain it better in the docstring I believe this comment is not needed